Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Globalization;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886678)